DuckDuckGoTests/NetworkProtectionStatusViewModelTests.swift

```diff
@@ -204,6 +204,6 @@ final class NetworkProtectionStatusViewModelTests: XCTestCase {
             condition()
         }
         let expectation = XCTNSPredicateExpectation(predicate: predicate, object: nil)
-        wait(for: [expectation], timeout: 5)
+        wait(for: [expectation], timeout: 20)
```

Comment:
Now that the simulator issue has been fixed, we can consider reverting these timeout changes. Hopefully they'll no longer be necessary.